From f5c9fb1234ada1e4d3f060d823dd2e754b97feb2 Mon Sep 17 00:00:00 2001
From: Ash Blake <ash.blake@tutanota.com>
Date: Sat, 17 Apr 2021 13:40:13 +0200
Subject: [PATCH] merlin: rootdir: init.mt6768: fix ownership for
 /dev/video{0,1}

Mediacodec did not have access and video recording didn't work.
---
 rootdir/etc/init.mt6768.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/etc/init.mt6768.rc b/rootdir/etc/init.mt6768.rc
index 24f91b0..25e5403 100644
--- a/rootdir/etc/init.mt6768.rc
+++ b/rootdir/etc/init.mt6768.rc
@@ -373,9 +373,9 @@ on post-fs-data
     chmod 0660 /dev/vcu
     chown media system /dev/vcu
     chmod 0660 /dev/video0
-    chown media system /dev/video0
+    chown mediacodec system /dev/video0
     chmod 0660 /dev/video1
-    chown media system /dev/video1
+    chown mediacodec system /dev/video1
     mkdir /data/vendor/vcodec 0770 media system
     mkdir /data/vendor/vcodec/log 0770 media system
 
